src/course-search/SearchBox.jsx

```diff
@@ -26,6 +32,11 @@ export const SearchBoxBase = ({
   const handleSubmit = (searchQuery) => {
     dispatch(setRefinementAction(QUERY_PARAM_FOR_SEARCH_QUERY, searchQuery));
     dispatch(deleteRefinementAction(QUERY_PARAM_FOR_PAGE));
+    if (trackingName) {
+      sendTrackEvent(`${SEARCH_EVENT_NAME_PREFIX}.${trackingName}.catalog_search`, {
```

Comment:
Might suggest renaming to ``${SEARCH_EVENT_NAME_PREFIX}.${trackingName}.catalog_search.query_submitted`` or similar so that we could expand on the `catalog_search` namespace with future events that aren't directly related to search submission.

Reply:
I see what you mean, I like it.. I can see `catalog_search.filter_applied` etc too